.theia/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive",
+    "files.exclude": {
+        "**/.classpath": true,
+        "**/.project": true,
+        "**/.settings": true,
+        "**/.factorypath": true
+    }
+}
```

Comment:
I don't think this needs to be part of the PR. you can remove it. I think this is some settings for the cloud editor